### DISCUSSION TITLE
Highlight opening directive from closing directive

### DISCUSSION THREAD

**Discussion by cesarreyes3** [haidubogdan/netbeans-php-blade-plugin#68] [Ideas]
_From: https://github.com/haidubogdan/netbeans-php-blade-plugin/issues/67#issuecomment-2288804758_ 

it is something minimal, if it complicates everything it would be better to ignore it,

On custom directives, if you click/select an opening directive, closing directive gets highlighted
![image](https://github.com/user-attachments/assets/e0ce6a35-9e27-4ace-89b2-f58c1d7f688d)

But if you click/select a closing directive, opening directive does nothing
![image](https://github.com/user-attachments/assets/3846bfd7-3792-43b0-9c5b-a4829d35ac5a)

In large blocks it's difficult to find the opening directive from closing directive 

```blade
@modal()
@card()
@card()
@card()
@endcard
@endcard
@endcard            
@endmodal
```

**Comment by cesarreyes3** [comment]
https://github.com/haidubogdan/netbeans-php-blade-plugin/releases/tag/nbpr1.0.9

It works, thanks

**Reply by haidubogdan** [reply]
With pleasure :)
